doc/tvm.tex

```diff
@@ -2225,7 +2226,6 @@ \section*{Introduction}
 \item {\tt F802} --- {\tt BUYGAS} ($x$ -- ), computes the amount of gas that can be bought for $x$ nanograms, and sets $g_l$ accordingly in the same way as {\tt SETGASLIMIT}.
 \item {\tt F804} --- {\tt GRAMTOGAS} ($x$ -- $g$), computes the amount of gas that can be bought for $x$ nanograms. If $x$ is negative, returns 0. If $g$ exceeds $2^{63}-1$, it is replaced with this value.
 \item {\tt F805} --- {\tt GASTOGRAM} ($g$ -- $x$), computes the price of $g$ gas in nanograms.
-\item {\tt F806} --- {\tt GASREMAINING} ( -- $g$), returns the remaining gas $g_r$.
```

Comment:
why?

Reply:
this code is not in the reference specification, a sign that it is missing, and this is apparently where I am wrong